docs/web/diff.md

```diff
@@ -0,0 +1,902 @@
+
+![image](https://github.com/Szelethus/codechecker/assets/23276031/e11790b7-9b6c-418e-b217-6d5b00108e26)
+
+:warning: Note: It is possible for a report to not be outstanding in any of the "baseline" or the "newline" sets. This would have happened if we added the source code suppression for the report made by `deadcode.DeadStores` in the baseline as well. Such reports are abscent from the results of diff, regardless of whether `--new`, `--resolved` or `--unresolved` is used.
+
+### Using `CodeChecker cmd diff`
+:warning: Note: Below, only local-local diffs are shown, the remote-remote, local-remote/remote-local diffs are analogous to what's shown [above](#using-codechecker-cmd-diff). Also, we only show the invocation from the command line, not the web GUI, which we also displayed [above](#using-the-web-gui). The result of comparing runs on the command line is the same as comparing them on the web GUI. Let us know if you see any divergence!
+
+* Show outstanding reports that appeared from first analysis to the next (via `--new`):
+  ```sh
+  CodeChecker cmd diff --basename ./test_report_dir --newname ./test_report_dir_src_suppressed/ --new
+  ```
+  Expected output:
+  ```
+  ----==== Checker Statistics ====----
+  ---------------------------------------------------
+  Checker name         | Severity | Number of reports
+  ---------------------------------------------------
+  core.NullDereference | HIGH     |                 1
+  ---------------------------------------------------
+  ----=================----
+  ```
+* Show outstanding reports that disappeared from first analysis to the next (via `--resolved`): 
+  ```sh
+  CodeChecker cmd diff --basename ./test_report_dir --newname ./test_report_dir_src_suppressed/ --resolved
+  ```
+  Expected output:
+  ```
+  ----==== Checker Statistics ====----
+  --------------------------------------------------
+  Checker name        | Severity | Number of reports
+  --------------------------------------------------
+  core.DivideZero     | HIGH     |                 1
+  deadcode.DeadStores | LOW      |                 1
+  --------------------------------------------------
+  ----=================----
+  ```
+* Show outstanding reports that remained from first analysis to the next (via `--unresolved`): 
+  ```sh
+  CodeChecker cmd diff --basename ./test_report_dir --newname ./test_report_dir_src_suppressed/ --unresolved
+  ```
+  Expected output (no checkers are seen):
+  ```
+  ----======== Summary ========----
+  ---------------------------------------------
+  Number of processed analyzer result files | 0
+  Number of analyzer reports                | 0
+  ---------------------------------------------
+  ----=================----
+  ```
+
+## Review status rules and diffs
```

Comment:
Refer to the use cases introduced in the introduction section.

E.g.

If you whish to store the suppressions on the server, you can use review status rules. With the review status rule you can set all instances of report type (reports with the same hash) e.g. false positive. This feature makes it easy to mark a report false positive in all runs at the same time.